services/github/test/pr-allowed_test.js

```diff
@@ -1,151 +0,0 @@
-const debug = require('debug')('test');
-const helper = require('./helper');
-const assert = require('assert');
-const prAllowed = require('../src/pr-allowed');
-
-suite('allowPullRequests', function() {
```

Comment:
Did these tests get lost in the shuffle?

Reply:
We test the PR-allowed checking logic in handlers tests already. I can add more test cases if you think there's anything missing